notebooks/407-person-tracking-webcam/407-person-tracking-paddle.ipynb

```diff
@@ -0,0 +1,915 @@
+   "id": "73d7e127-7ea2-4149-b860-c2fc22a79651",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the function to download mot_ppyoloe_s_36e_pipeline model from PaddleDetection resources.\n",
+    "\n",
+    "\n",
+    "def run_model_download(model_url, model_file_path):\n",
+    "    \"\"\"\n",
+    "    Download pre-trained models from PaddleDetection resources\n",
+    "\n",
+    "    Parameters:\n",
+    "        model_url: url link to pre-trained models\n",
+    "        model_file_path: file path to store the downloaded model\n",
+    "    \"\"\"\n",
+    "    model_name = model_url.split(\"/\")[-1]\n",
+    "\n",
+    "    if model_file_path.is_file():\n",
+    "        print(\"Model already exists\")\n",
+    "    else:\n",
+    "        # Download the model from the server, and untar it.\n",
+    "        print(\"Downloading the pre-trained model... May take a while...\")\n",
+    "\n",
+    "        # Create a directory.\n",
+    "        os.makedirs(\"model\", exist_ok=True)\n",
+    "        urllib.request.urlretrieve(model_url, f\"model/{model_name} \")\n",
+    "        print(\"Model Downloaded\")\n",
+    "        print(f\"model/{model_name} \")\n",
+    "        file = zipfile.ZipFile(f\"model/{model_name} \")\n",
+    "        res = file.extractall(\"model\")\n",
+    "        file.close()\n",
+    "        if not res:\n",
+    "            print(f\"Model Extracted to {model_file_path}.\")\n",
+    "        else:\n",
+    "            print(\"Error Extracting the model. Please check the network.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9149e74-daf5-4786-a163-998990e24365",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Download the Model\n",
+    "\n",
+    "You can also use https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip (184mb) to get better results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "75312f82-47e0-47ac-bad4-f702deea025d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# A directory where the model will be downloaded.\n",
+    "\n",
+    "model_url = (\n",
+    "    \"https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip\"\n",
+    ")\n",
+    "model_file_path = Path(\"model/mot_ppyoloe_s_36e_pipeline/model.pdmodel\")\n",
+    "\n",
+    "run_model_download(model_url, model_file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e11303f-8fbc-4d03-8b9e-b41ba1265ec2",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Preprocessing Functions for Pedestrian counting\n",
+    "\n",
+    "### load preprocess config \n",
+    "\n",
+    "All PaddlePaddle Pre-trained model downloaded from PaddleDetection contains `infer_cfg.yml` file which All image preprocessing configurations are saved. \n",
+    "\n",
+    "Using the code given by PaddleDetection, we can load `infer_cfg.yml` file and complete the preprocess easily"
```

Comment:
Does all parameters got from ```infer_cfg.yml```` will used in this example? otherwise we can simplify this function, or just read the key parameter.  We have a example to implement PPYoloV:
https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/203-meter-reader